src/components/Sidebar/Sidebar.jsx

```diff
@@ -1,9 +1,11 @@
 import "../Sidebar/Sidebar.css";
 import { Link } from "react-router-dom";
+import { useData } from "../../Context/data-context";
 export const Sidebar = () => {
+  const {sidebar}=useData()
   return (
     <div>
-      <aside style={{margin:"0"}} className="video-library-sidebar">
+      <aside style={{position:"fixed",left:sidebar,transition:"all 0.5s ease-in-out"}}  className="video-library-sidebar">
```

Comment:
can avoid inline styling and add these styles in class

Reply:
ok